src/app/services/raiden.service.ts

```diff
@@ -349,25 +349,14 @@ export class RaidenService {
             );
     }
 
-    public registerToken(tokenAddress: string): Observable<UserToken> {
+    public registerToken(tokenAddress: string): Observable<any> {
         return this.http
             .put(`${this.raidenConfig.api}/tokens/${tokenAddress}`, {})
             .pipe(
-                map(() => {
-                    const userToken = this.getUserToken(tokenAddress);
-                    if (userToken === null) {
-                        throw new Error(
-                            `No contract on address: ${tokenAddress}`
-                        );
-                    }
-                    return userToken;
-                }),
-                tap((userToken: UserToken) => {
+                tap(() => {
```

Comment:
```suggestion
                map(() => null),
                tap(() => {
```
for the void/ignore to work you also have to map to null otherwise the TypeScript compiler will complain about a type missmatch.